kudulite/Dockerfile

```diff
@@ -65,12 +65,14 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 # Use Dynamic Install SDK feature from Oryx build
 # Trim off all unnecessary SDKs and let them lazily loaded
 RUN rm -rf /opt/python /opt/hugo /opt/php /opt/php-composer \
-    # We still need the fundamental ones when building the images nodejs 9, npm 6 and dotnet 2.2
-    # TODO: This will be upgraded to dotnet 3.1 soon
+    # We still need the fundamental ones when building the images nodejs 9, npm 6 and dotnet
     && find /opt/nodejs -mindepth 1 -maxdepth 1 -type d,l -not -name "9*" | xargs rm -rf \
-    && find /opt/npm -mindepth 1 -maxdepth 1 -type d,l -not -name "6*" | xargs rm -rf \
-    && find /opt/dotnet/runtimes -mindepth 1 -maxdepth 1 -type d,l -not -name '2*' | xargs rm -rf \
-    && find /opt/dotnet/sdks -mindepth 1 -maxdepth 1 -type d,l -not -name '2*' | xargs rm -rf
+    && find /opt/npm -mindepth 1 -maxdepth 1 -type d,l -not -name "6*" | xargs rm -rf
+
+# There's an exising bug in dotnet version detector in oryx image where it will resolve dotnet 2.2 into 2.2.8
+# But the image fails to resolve 2.2.8 runtime, thus, adding a symbolic link as temporary mitigation
+RUN ln -s /opt/dotnet/runtimes/2.2.7 /opt/dotnet/runtimes/2.2.8
```

Comment:
Why do we need to link 2.2.7 to 2.2.8?

Reply:
Because the current oryx build framework detector always resolves 2.2 into 2.2.8 as the oryx build image does not provide it. Kiran is investigating the issue. This is temporary mitigation for netcoreapp2.2 function apps.